rts/Sim/Projectiles/WeaponProjectiles/FlameProjectile.cpp

```diff
@@ -62,10 +62,7 @@ void CFlameProjectile::Update()
 	}
 
 	UpdateInterception();
-
-	radius = radius + weaponDef->sizeGrowth;
-	sqRadius = radius * radius;
-	drawRadius = radius * weaponDef->collisionSize;
+	drawRadius = radius + weaponDef->sizeGrowth;
```

Comment:
Now it won't grow because both components of the sum are constant, and also it wasn't supposed to rely on the physical `radius`.